services/housetables/src/main/resources/schema.sql

```diff
@@ -5,6 +5,7 @@ CREATE TABLE IF NOT EXISTS user_table_row (
                          version             BIGINT            NOT NULL,
                          metadata_location   VARCHAR (512)     ,
                          storage_type        VARCHAR (128)     DEFAULT 'hdfs' NOT NULL,
+                         creation_time       BIGINT            DEFAULT NULL,
```

Comment:
why are we using bigint here but last_modified_time is timestamp?

Reply:
i think we should be consistent between them to help prevent potential logic/user issues with querying and inserting two different time types and converting them correctly

Reply:
I believe this field is bigint as epoch time is used for creation time. Is there a specific reason for using epoch time?

Reply:
Default value will be 0 BTW.

Reply:
The schema.sql is not the used for holdem or war. Not exactly sure why we don't update the file? @autumnust 
Also, I'm using long type because the `creationTime` and `lastModifiedTime` in table properties have the long type. I wanted to keep simple to avoid transformation.

Reply:
Can we explore if epochTime can be converted to Timestamp type while mapping and inserting to table manually. Agree with @cbb330 that `creation_time` and `last_modified_time` consistency could be crucial.

Reply:
Me preferring long / bigint for all timestamp related columns for the reason of:  don't need to deal with the complexities of time zone, day light saving etc. that tends to be differently handled in different languages / environment. On the hand, almost all query engines provides good library to convert long time into a timestamp representation in the way you want.

Reply:
I have no answer why inconsistency exists right now ... but let's agree on what's the consistent type we wanted first.